auth_helper.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pathlib
+
+import json5
+import sys
+import requests
+import msal
+
+class AuthHelper:
+    tenantId = os.environ["tenant_id"]
```

Comment:
This is setting class variables, not instance variables. These should also be taken in the constructor.

Reply:
Resolved.